src/applications/claims-status/components/StemDeniedDetails.jsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import moment from 'moment';
+
+const StemDeniedDetails = ({
+  deniedAt,
+  isEnrolledStem,
+  isPursuingTeachingCert,
+}) => {
+  const date = moment(deniedAt).format('MMM D, YYYY');
+
+  return (
+    <>
+      <h1>Your Edith Nourse Rogers STEM Scholarship Application</h1>
+      <div className="vads-u-background-color--primary-alt-lightest vads-u-padding--2 vads-u-margin-bottom--3">
+        <h3 className="claims-alert-header vads-u-font-size--h4">
+          Your application was denied on {date}.
+        </h3>
+      </div>
+      <h2 className="vads-u-font-family--sans vads-u-margin-y--0">
+        Why we denied your claim
+      </h2>
+      <p className="vads-u-margin--0">
+        You must meet all 3 of the eligibility criteria to be considered for the
+        Rogers STEM Scholarship.
+      </p>
+      <h3 className="vads-u-font-family--sans vads-u-margin-bottom--0">
+        You did not meet these criteria for the Rogers STEM Scholarship:
+      </h3>
+      <ul className="stem-ad-list">
```

Comment:
Do these lists need some left margin? In the screenshot it appears as if the bullet points aren't aligned evenly to the headers.

Reply:
This is a design specific thing, I went ahead and added an additional `2px` in the latest commit to get these to align to our design prototype as closely as possible.